ansible/roles/chroma/templates/chroma.service.j2

```diff
@@ -0,0 +1,32 @@
+# {{ ansible_managed }}
+
+[Unit]
+Description=Run the Chroma basckend
+OnFailure=failure-notificator@%n.service
+
+[Service]
+Type=simple
+User=chroma
+Group=chroma
+Restart=always
+WorkingDirectory=/var/www/chroma/server
+
+Environment=HTTP_PORT=34971
```

Comment:
we should probably move everything to .env.j2, having a split config is annoying. We should also check if the file must be a hidden file, because that is quite annoying to work with as IDEs will hide the file by default.